.travis.yml

```diff
@@ -13,7 +13,8 @@ install:
 script:
   - make CI=1
   - make test
-  - goveralls -service=travis-ci
+  - ./coverage.sh
```

Comment:
Can we move this (potentially along with other travis-related files) to a separate dir, e.g. "ci", so we don't pollute the root dir with too many CI related files?